contracts/Deposit.sol

```diff
@@ -0,0 +1,84 @@
+pragma solidity ^0.4.19;
+
+contract AssetManager{
+    /*
+    * Standard interface contract for deploying SimpleDeposit contracts
+    */
+    
+    address public creator;
+    address[] public deposits;
+    uint public minimum_deposit_threshold = 0.2 ether;
+    uint public minimum_fee = 0.1 ether;
+    
+    function AssetManager() public{
+        creator = msg.sender;
+    }
+    
+    function createDeposit(address assetOwner, uint fee, uint daysUntilExpiry) payable public{
+        require(msg.value >= minimum_deposit_threshold && msg.value > fee * 2 ether);
+        require(fee * 1 ether > minimum_fee);
+        require(daysUntilExpiry >= 1);
+        address newDeposit = (new SimpleDeposit).value(msg.value)(msg.sender, assetOwner, fee, daysUntilExpiry);
+        deposits.push(newDeposit);
+    }
+    
+    // get a count on how many SimpleDeposit Contracts have been deployed using this
+    function getDepositCount() public view returns(uint){
+        return deposits.length;
+    }
+}
+
+contract SimpleDeposit{
+    
+    address public depositor;
+    address public assetOwner;
+    bytes32 private assetInfo;
+    uint public fee;
+    uint public timeOfCreation;
+    uint public validFor;
+    
+    modifier onlyDepositor(){
+        require(msg.sender == depositor);
+        _;
+    }
+    
+    modifier onlyAssetOwner(){
+        require(msg.sender == assetOwner);
+        _;
+    }
+    
+    function SimpleDeposit(address _depositor, address _assetOwner, uint _fee, uint _validFor) public payable{
+        require(msg.value > 2*_fee);
+        depositor = _depositor;
+        assetOwner = _assetOwner;
+        fee = _fee;
+        timeOfCreation = now; 
```

Comment:
I got a warning on compile:
"browser/SimpleDeposit.sol:55:26:use of "now": "now" does not mean current time. Now is an alias for block.timestamp. Block.timestamp can be influenced by miners to a certain degree, be careful."

Just FYI.